### PR TITLE
Translation to PT-PT

### PR DESCRIPTION
**Pull request by vb-03**

### CODE REVIEW

**Comment by OhMyGuus** [comment]
Can u translate:
"meetings_only_warning2": "Talking in meetings only enabled.",
"ghost_only_warning2": "Ghost can talk only enabled.",

Since they where missing in the version u translated.

**Comment by vb-03** [comment]
Done

**Review by vb-03 — COMMENTED** [review]
~Isn't "translationPTBR" duplicate?~

File: src/renderer/language/languages.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import translationEN from '../../../static/locales/en/translation.json'
 import translationFR from '../../../static/locales/fr/translation.json'
 import translationES from '../../../static/locales/es/translation.json'
 import translationPTBR from '../../../static/locales/pt-BR/translation.json'
+import translationPTBR from '../../../static/locales/pt-PT/translation.json'
```
Comment (vb-03) [inline src/renderer/language/languages.ts:5]: Here it should be PTPT

**Comment by MatadorProBr** [comment]
> Isn't "translationPTBR" duplicate?

Resolved

File: src/renderer/language/languages.ts (hunk 1 above)
Comment (MatadorProBr) [reply src/renderer/language/languages.ts:5]: oh

Comment (MatadorProBr) [reply src/renderer/language/languages.ts]: resolved

**Comment by vb-03** [comment]
I believe everything is ok now

**Comment by MatadorProBr** [comment]
I will accept the pull request after checking AppVeyor

**Comment by vb-03** [comment]
> I will accept the pull request after checking AppVeyor
> ???

My bad, closed it by accident

**Comment by MatadorProBr** [comment]
I just noticed some flaws

**Comment by MatadorProBr** [comment]
Now you can delete the branch

**Pull request merged**